src/pkg/project/manager.go

```diff
@@ -21,6 +21,11 @@ import (
 	"github.com/goharbor/harbor/src/common/models"
 )
 
+var (
+	// Mgr is the global project manager
+	Mgr = New()
```

Comment:
Update the references of Global project manager?

Reply:
Not covered in this PR, the project manager and repository manager should be refactored later

Reply:
so we don't need to add this?

Reply:
Updated